tests/urlpatterns_reverse/tests.py

```diff
@@ -429,6 +430,17 @@ def test_namespaced_view_detail(self):
         self.assertTrue(resolver._is_callback('urlpatterns_reverse.nested_urls.View3'))
         self.assertFalse(resolver._is_callback('urlpatterns_reverse.nested_urls.blub'))
 
+    def test_populate_concurrency(self):
+        """
+        RegexURLResolver._populate() can be called concurrently (#26888).
```

Comment:
I think my previous comment got lost: Could you add "... concurrently but not more than once per thread ..." please.